src/controllers/Lane.js

```diff
@@ -31,10 +31,7 @@ class Lane extends Component {
       this.setState({loading: true})
       const nextPage = currentPage + 1
       onLaneScroll(nextPage, this.props.id).then(moreCards => {
-        if (!moreCards || moreCards.length === 0) {
-          // if no cards present, stop retrying until user action
-          node.scrollTop = node.scrollTop - 100
```

Comment:
Why did you remove this line? It has beed added to fix this issue - https://github.com/rcdexta/react-trello/issues/54

Reply:
it's is the reason for the problem, 
when there is no card, so on the bottom of the lane your scroll jumps up by 100, and the last card stays covered. so you need to scroll down very slowly or make your cards bigger.
so this line did not fix anything.
 if there isn't any card why make scroll move?

Reply:
@dapi I second what @gevhambarzumyan93 says. It's the whole reason why this is happening. I don't know why it was added back then but at least right now it doesn't seem to have any use anymore. I tested his changes locally too and it's all working smooth as it should be.

Reply:
Guys, thank you!